modules/search/instant-search/lib/api.js

```diff
@@ -103,10 +103,13 @@ const filterKeyToEsFilter = new Map( [
 	],
 ] );
 
-function buildFilterObject( filterQuery ) {
-	if ( ! filterQuery ) {
+function buildFilterObject( filterQuery, adminQueryFilter ) {
+	if ( ! filterQuery && ! adminQueryFilter ) {
 		return {};
 	}
+	if ( ! filterQuery ) {
+		return adminQueryFilter;
```

Comment:
Shouldn't this be the following?

```js
return { bool: { must: adminQueryFilter } }
```

Reply:
As long as the user created a proper filter then it should just work. No need to wrap it in a separate bool clause.